src/doc/trpl/README.md

```diff
@@ -34,11 +33,11 @@ is the first. After this:
 [gl]: glossary.html
 [ar]: academic-research.html
 
-After reading this introduction, you’ll want to dive into either ‘Learn Rust’
-or ‘Syntax and Semantics’, depending on your preference: ‘Learn Rust’ if you
-want to dive in with a project, or ‘Syntax and Semantics’ if you prefer to
-start small, and learn a single concept thoroughly before moving onto the next.
-Copious cross-linking connects these parts together.
+After reading this introduction, we’ll want to dive into either ‘Learn Rust’ or
+‘Syntax and Semantics’, depending on our preference: ‘Learn Rust’ if we want to
+dive in with a project, or ‘Syntax and Semantics’ if we prefer to start small,
+and learn a single concept thoroughly before moving onto the next. Copious
+cross-linking connects these parts together.
```

Comment:
So, while I'm generally super in favor of moving EVERYTHING to 'we', I think that this section should stay as-is. In this case, we are addressing the reader directly, they're the one making this choice. I think from here and above, 'you' is fine.